src/xrCore/xr_ini.h

```diff
@@ -83,7 +83,7 @@ class XRCORE_API CInifile
     void save_as(IWriter& writer, bool bcheck = false) const;
     void set_override_names(bool b) noexcept { m_flags.set(eOverrideNames, b); }
     void save_at_end(bool b) noexcept { m_flags.set(eSaveAtEnd, b); }
-    pcstr fname() const noexcept { return m_file_name; };
+    pcstr fname() const { return m_file_name; };
```

Comment:
Ну я же попросил закомментировать, а не удалять...
```suggestion
    pcstr fname() const /*noexcept*/ { return m_file_name; };
```